### PR TITLE
asterisk_mbox_server update

### LINKED ISSUE #255: Mailbox server broken
Just a reminder for myself:

The asterisk mailbox server package seems broken. Should be a easy fix.

### PR DESCRIPTION
**Pull request by miguelangel-nubla**
Fixes #255

### CODE REVIEW

File: asterisk/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,6 @@ FROM ghcr.io/hassio-addons/debian-base:6.2.7 AS base
 SHELL [ "/bin/bash", "-euxo", "pipefail", "-c" ]
 
 # https://pypi.org/project/asterisk_mbox_server/#history
```
Comment (TECH7Fox) [inline asterisk/Dockerfile:6]: Best to move the comment as well then.

**Comment by TECH7Fox** [comment]
Looks good now. Let's wait to see if the tests succeed now before merging.

**Comment by TECH7Fox** [comment]
And can you also change the version to 4.0.4? Since we never actually released it. Also don't forget adding the patch fix to the changelog.

**Comment by TECH7Fox** [comment]
Build seems to be working. Will merge.

Thanks for your work @miguelangel-nubla!

**Comment by miguelangel-nubla** [comment]
Lint did not like two-step install... https://github.com/TECH7Fox/asterisk-hass-addons/actions/runs/5148445967

**Comment by TECH7Fox** [comment]
Yeah I saw it, didn't think about that. Will fix it.

**Pull request merged**